extensions/2.0/Vendor/EXT_feature_metadata/schema/featureClass.property.schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Property class",
+  "type": "object",
+  "description": "A property class.",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+      "name": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "The name of the property, e.g. for display purposes. The string is UTF-8 encoded."
+      },
+      "description": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "The description of the property. The string is UTF-8 encoded."
+      },
+      "valueType": {
+          "type": "string",
+          "enum": ["BOOLEAN", "INT8", "UINT8", "INT16", "UINT16", "INT32", "UINT32", "INT64", "UINT64", "FLOAT16", "FLOAT32", "FLOAT64", "STRING", "BINARY"],
```

Comment:
Interesting idea about aliases...

There's no difference between `BINARY` and `VARIABLE_SIZE_ARRAY[UINT8]` as far as how the data is stored. But when someone thinks of a binary blob do they think of an array of uint8's? I don't know.

Similar question, is `STRING` a `VARIABLE_SIZE_ARRAY[UINT8]` too? From https://arrow.apache.org/docs/format/Columnar.html#terminology:

> Logical type: An application-facing semantic value type that is implemented using some physical layout. For example, Decimal values are stored as 16 bytes in a fixed-size binary layout. Similarly, strings can be stored as List<1-byte>. A timestamp may be stored as 64-bit fixed-size layout.

Reply:
Do we plan to support unicode for String? If we do, maybe it should have another field to indicate the encoding type

Reply:
@baothientran yes, the current wording specifies that STRING is encoded as UTF-8

I think that's the key point here, STRING is a variable size array of bytes, _but_ it's UTF-8 aware where an arbitrary BINARY chunk would not be.

Reply:
@ptrgags Ah I see it now. Yeah I agree that STRING should be a separate type as it is now. For the BINARY type, I think it is the convention to indicate array of UINT8 (at least in C or C++). Should we remove it to make it simpler?

Reply:
I talked about this with @lilleyse, we're thinking of doing this:

* STRING should stay as-is because of the UTF-8 handling
* BINARY is the only alias we allow, it's an alias for `VARIABLE_SIZE_ARRAY[UINT8]`. The purpose of the alias is that it gives semantic meaning that "these bytes are a binary blob"
* Everything else with arrays stays as-is.

@baothientran does this sound reasonable to you?

Reply:
yep that looks good to me